src/main/kotlin/no/nav/syfo/services/UpdateInfotrygdService.kt

```diff
@@ -535,7 +536,7 @@ class UpdateInfotrygdService {
                     behandlingstype = "ANY"
                     mappeId = 1
                     aktivDato = DateTimeFormatter.ISO_DATE.format(LocalDate.now())
-                    fristFerdigstillelse = DateTimeFormatter.ISO_DATE.format(LocalDate.now())
+                    fristFerdigstillelse = DateTimeFormatter.ISO_DATE.format(finnFristForFerdigstillingAvOppgave(LocalDate.now()))
```

Comment:
Her må du ha now() pluss fristen, hvis ikke blir det samme feil som nå, bare justert for helg ;)